tutorials/v7/draw_rh1.cxx

```diff
@@ -45,8 +45,13 @@ void draw_rh1() {
 
    // Create a canvas to be displayed.
    auto canvas = RCanvas::Create("Canvas Title");
-   canvas->Draw(pHist)->Line().SetColor(RColor::kRed);
-   canvas->Draw(pHist2)->Line().SetColor(RColor::kBlue);
+   auto draw1 = canvas->Draw(pHist);
+   draw1->Line().SetColor(RColor::kRed);
+   draw1->Line().SetWidth(2);
```

Comment:
Or `draw1->Line().SetColor(RColor::kRed).SetWidth(2);`

Reply:
Grate, if it works this way! Examples will be change many times anyway.